src/main/webapp/app/sharing/sharing.component.html

```
@@ -0,0 +1,121 @@
+                                            <th></th>
+                                            <th class="d-none d-md-table-cell" jhiTranslate="artemisApp.sharing.exerciseToImport">Exercise to import</th>
+                                        </tr>
+                                    </thead>
+                                    <tbody>
+                                        @for (exercise of shoppingBasket.exerciseInfo; track exercise; let exercisePos = $index) {
+                                            <tr>
+                                                <td>
+                                                    <input
+                                                        id="{{ exercise.project.project_id }}"
+                                                        [attr.aria-label="'Select exercise: ' + exercise.metadata.title"
+                                                        [(ngModel)]="sharingInfo.selectedExercise"
```

HTMLHint (1.5.0)

[error] The attribute name of [ [(ngModel)] ] must be in lowercase.

(attr-lowercase)